musicrs/util/youtubeIdToLink.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+def youtubeIdToLink(basePath):
+
+  baseUrl = 'https://www.youtube.com/watch?v='
+
+  entries = os.listdir(basePath)
+  for entry in entries:
+    f=open(os.path.join(basePath,entry), "r")
+    if f.mode == 'r':
+      contents =f.readlines()
+      for youtubeId in contents: 
+        if not '# new artist:' in youtubeId: 
```

Comment:
It should ignore all the lines starting with #, not just `# new artist` because we might use other datasets that have lines with `#` followed by something else.

Reply:
:ok: